HHS/Head-Start-TTADP
# [TTAHUB-1857] Update event permissions on BE and FE

## Description of change

Only a user with write permission and a assigned event role (poc, collab, or owner) can create or edit an event and session.

## How to test

Create a user who has write permission but is NOT assigned and a event role. They should only be able to view.

Create a user who has write permission and IS assigned a role. They should be able to create, read, update and delete that event and its sessions.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-0=1857


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
